netlify.toml

```diff
@@ -109,6 +109,7 @@ REACT_APP_SERVER_URL = "https://goodserver.gooddollar.org/"
 REACT_APP_GUN_PUBLIC_URL = "https://goodgun-nvme.gooddollar.org/gun"
 REACT_APP_NETWORK = "production"
 REACT_APP_SKIP_EMAIL_VERIFICATION = "false"
+REACT_APP_SKIP_MOBILE_VERIFICATION = "true"
```

Comment:
if i remember - we also have to set the corresponding env flag server-side. could you please check that ?

Reply:
it is already like that on server side. (today we have the "skip" option, after user requests sms code 2 times)